paddle/fluid/inference/tests/api/analyzer_transformer_tester.cc

```diff
@@ -214,23 +214,25 @@ TEST(Analyzer_Transformer, fuse_statis) {
 }
 
```

Comment:
Please add `TODO(yihuaxu): Disable compare and compare_mkldnn temporary, see https://github.com/paddlePaddle/Paddle/issues/16316 for details.`

Reply:
Done